web/themes/custom/cloudy/src/layouts/drawer/drawer.md

```diff
@@ -0,0 +1,20 @@
+# Drawer
+
+## Props
+
+- `content`: content of the drawer
+- `position`: left (default), right - Does the drawer sit on the left or right. On larger screen, is it a left or right column.
+- `open_text`: string - defaults to "Menu"
+- `open_info`: string - defaults to "Expand side filters"
```

Comment:
My only issue with this language is that it is directional. If I'm using a screen reader, "side" doesn't have the correct meaning. Maybe just update to say "Expand filters" or maybe get more descriptive like "Expand filters have less results on this page".

What do you think?